e2e/scripts/validate-generated-project.js

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright (c) 2023, Salesforce, Inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+const { program, Argument } = require("commander");
+const { diffArrays } = require("./utils.js");
+const fs = require("fs");
+const config = require("../config.js");
+const path = require("path");
+
+const validateGeneratedArtifacts = async (project) => {
```

Comment:
This is a good start, am I correct that I don't see any Playwright code in this first PR. I assume that's a follow up?

Reply:
Yes. This is mostly the generator tests and deploying to MRT so that Playwright has a definite target to run on. That'll be a follow-up PR